health-monitor/src/index.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import dotenv from 'dotenv';
+import express from 'express';
+import fetch from 'node-fetch';
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+dotenv.config();
+
+const app = express();
+
+app.set('host', '0.0.0.0');
+app.set('port', process.env.PORT || 8034);
+
+export interface HealthCheckResult {
+	authServerStatus: number;
+	hasuraServerStatus: number;
+	chainDbWatcherServerStatus: number;
+	reactServerStatus: number;
+	latestBlockNumber: number;
+	prismaVersion: string;
+}
+
+async function getLatestBlockNumber (): Promise<number> {
+	const provider = new WsProvider(process.env.ARCHIVE_NODE_ENDPOINT);
+	const api = await ApiPromise.create({ provider });
+
+	const latestBlockNumber = (
+		await api.derive.chain.bestNumberFinalized()
```

Comment:
let's get the latest block rather than the finalized one. This one could stall and raise an error although we don't depend on finalized blocks any more.